app/api/routes.py

```
@@ -516,6 +517,58 @@ def delete_consult_session_route(
 
 
+@router.post(
+    "/api/v1/history/{session_id}/summary",
+    tags=["에이전트 상담"],
+    response_model=ConsultSessionSummaryRow,
+    summary="특정 상담 세션 요약",
+    description="특정 상담 세션이 끝난 후, 전체 상담 내용을 50자~120자 내로 요약하여 제공합니다.",
+)
+def create_consult_summary(
+    session_id: str,
+    db: Session = Depends(get_db),
```

Ruff (0.14.7)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)